scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -138,7 +141,8 @@ class PostInstallCmd(Cmd):
                 self.logger.warning("Cluster is already up...Exiting.\n")
                 return
         if self.k8:
-            config_part(self)
+            # Partitioning of metadata disk is not required for cortx-m0 release
```

Comment:
remove the code if not used

Reply:
Added comment

Reply:
Replace comment appropriately like "Nothing required for post_install stage in k8"